UnchainedLauncher.Core/Services/Mods/Registry/LocalModRegistry.cs

```diff
@@ -13,6 +15,8 @@ public LocalModRegistry(string registryPath) {
             RegistryPath = registryPath;
         }
 
+        public event Action? OnRegistryChanged;
```

Comment:
could this action accept ReleaseCoordinates as input, that way any registered callbacks can know specifically which release changed?